packages/types/src/index.ts

```diff
@@ -16,6 +16,7 @@ export type { KeyringPair } from '@polkadot/keyring/types'
 export * as SubscriptionPromise from './SubscriptionPromise.js'
 
 export * from './Attestation.js'
+export * from './AssetDid.js'
```

Comment:
Like what? They are taking from the [AssetDID](https://github.com/KILTprotocol/spec-asset-did] spec which in turns takes them from the [CAIP-2](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md) and [CAIP-19](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md) definitions.

Reply:
Does this look okay to you? Remember, there are many other things in the SDK beyond this topic.
```ts
import type { ChainId } from '@kiltprotocol/types'

Reply:
Would you rather namespace them? Or simply add a prefix for these types?

Reply:
Ok probably here's my limited knowledge of TS hitting. We use them in the return type of the `parse` function. Does that make them required to be exported? If not, then we can only export the `AssetDid`.

Reply:
In that case, I’d try to move them all in the AssetDid.ts and only export `AssetDid`

Reply:
Then we would have to move the `IPublicCredentialInput` type as well, because so far the `@kiltprotocol/types` package does not depend on any other package of ours. How do we deal with that?

Reply:
I prefixed them with the standard they refer to. I guess there's nothing more accurate than that 😄 https://github.com/KILTprotocol/sdk-js/pull/677/commits/63d4a0542d29bb702078ba1925fd24bf230e00e9#diff-e450783712ca158e57332108b7545b8eb5d7b47bf88ce541bcc60a0017097adfR11